Applications/Spire/Source/Ui/ScrollableListBox.cpp

```diff
@@ -28,6 +28,7 @@ ScrollableListBox::ScrollableListBox(ListView& list_view, QWidget* parent)
   auto layout = new QHBoxLayout();
   layout->setContentsMargins({});
   m_scroll_box = new ScrollBox(m_list_view);
+  m_scroll_box->setFocusPolicy(Qt::NoFocus);
```

Comment:
Is this correct? Why is it done here instead of in ```DropDownList```?

Reply:
I think the ScrollableListBox is the container of the ListView with scrollbars, where the ScrollBox is used to display list items out of display range functionally. The ListView is supposed to have focus to navigate by keyboard, while the ScrollBox should not get focus when the Tab is pressed. The ScrollBox's focus policy is Qt::StrongFocus, so it have to be set to Qt::NoFocus.